config/locales/server.en.yml

```diff
@@ -2327,6 +2327,7 @@ en:
     create_revision_on_bulk_topic_moves: "Create revision for first posts when topics are moved into a new category in bulk."
 
     errors:
+      invalid_css_color: "Invalid color."
```

Comment:
Should we be more descriptive / helpful here? Maybe something like

> Invalid color. Either hexadecimal value or a color name.

Reply:
The site setting description says:

> The accent color to be used as the background of some elements in HTML emails. Enter a color name ('red') or hex value ('#FF0000').

I noticed all the error messages are short and cut to the point. Do you still want me to make the error more descriptive?

Reply:
Yes, we should always strive for better error messages.